app/index.js

```diff
@@ -15,8 +15,9 @@ app.commandLine.appendSwitch('enable-ntlm-v2', config.ntlmV2enabled);
 app.commandLine.appendSwitch('try-supported-channel-layouts');
 if (process.env.XDG_SESSION_TYPE == 'wayland') {
 	console.log('INFO: Running under Wayland, switching to PipeWire...');
-	app.commandLine.appendSwitch('ozone-platform', 'wayland');
-	app.commandLine.appendSwitch('enable-features', 'WebRTCPipeWireCapturer;UseOzonePlatform');
+	app.commandLine.appendSwitch('enable-features', 'WebRTCPipeWireCapturer,UseOzonePlatform');
+	//Ozone-platform wayland seems to lead to window with no title bar or blank window
+	//app.commandLine.appendSwitch('ozone-platform', 'wayland');
```

Comment:
Just remove this commented code (and the comment). Git is pretty good at keeping history.
```suggestion
```